gradle.properties

```diff
@@ -1,2 +1,4 @@
 group = com.bugsnag
 version = 2.4.2
+bintray_user=your_user
+bintray_api_key=your_api_key
```

Comment:
If these are added here, will it override `~/.gradle/gradle.properties`? I'm sure nobody wants to commit their credentials either.

Reply:
thanks @kattrali - I've updated the global properties file instead.